spec/celluloid/supervision_group_spec.rb

```diff
@@ -58,5 +62,19 @@ class MyGroup < Celluloid::SupervisionGroup
 
       expect(supervisor[:example]).to be_a MyActor
     end
+
+    it "supports injections" do
+      group_klass = Class.new(Celluloid::SupervisionGroup) do
+        def foo
+          add(MyActor, as: :bar, injections: {
+            initialize: Proc.new { @name = :foo }
+          })
+        end
+      end
+      group = group_klass.run!
+      group.foo
+      sleep 0.01
```

Comment:
Side discussion -- I keep seeing these. But does anyone think it'd be better to have some sort of `actor.ready?` in any case where race conditions could happen? Ideally adding `group.ready?` here would block until `group` is ready, rather than all this `sleep x` popping up everywhere.